app/components/captcha_submit_button_component.rb

```diff
@@ -1,14 +1,15 @@
 # frozen_string_literal: true
 
 class CaptchaSubmitButtonComponent < BaseComponent
-  attr_reader :form, :action, :tag_options
+  attr_reader :form, :action, :button_options, :tag_options
```

Comment:
this is the `recaptcha_action` not the older link-or-button action right? What if we renamed to `recapcha_action` tomatch the `RECAPTCHA_ACTION` constant it gets called with?

Reply:
Yeah, it's the reCAPTCHA concept of "action", also documented with a reference link a couple lines below.

https://github.com/18F/identity-idp/blob/2dd7c9ce98d3a92e9e1c6eace4c0ae09529d5a85/app/components/captcha_submit_button_component.rb#L8

But sure, I think renaming `recaptcha_action` could be clearer / more consistent.

Reply:
Also, that link should probably reference the Enterprise documentation, which is a little more complete and matches the expected production behavior:

https://cloud.google.com/recaptcha-enterprise/docs/actions-website